lib/navigation/view.ts

```diff
@@ -80,6 +82,11 @@ interface ViewData {
 	 */
 	// eslint-disable-next-line no-use-before-define
 	loadChildViews?: (view: View) => Promise<void>
+
+	/**
+	 * The view actions.
+	 */
+	actions?: ViewAction[],
```

Comment:
So this are actions only shown for one view and only registered by that view?

I am not sure if this is a good decision, as this limits how we can use it later, with the current API we noticed it is extremely flexible and we could reuse it for implementing a lot of new features not originally planned.
So I am not sure this is the best way here.

What I mean is:
Registering actions for a single view is already possible with the `enabled` prop where we can filter by view id.
But what is missing is an API to registering actions on the filelist itself and not a file.

So maybe it would be better to allow any app to register "file list actions", similar to the file actions.
This could then also be used for more use cases than this approach, as this one is limited to only the app that registers the view.